cpp/include/cuspatial/detail/utility/zero_data.cuh

```diff
@@ -19,6 +19,8 @@
 
 #include <cuspatial/traits.hpp>
 
+#include <rmm/cuda_stream_view.hpp>
+
```

Comment:
zero_data_async does take a cuda stream view as input so I think it's fair to add this dependency. We need this because all previous usage has cuda_stream_view.hpp included prior to this file. But this PR cpp/tests/range/multilinestring_range_test.cu includes this file and previous usage doesn't include the stream view dependency.